### PR TITLE
Add footnote clarifying param/FLOP counts after `model.fuse()`

### LINKED ISSUE #23593: Question: YOLOv26 parameter count mismatch vs docs
### Search before asking

- [x] I have searched the Ultralytics YOLO [issues](https://github.com/ultralytics/ultralytics/issues) and [discussions](https://github.com/orgs/ultralytics/discussions) and found no similar questions.

### Question

Hi Ultralytics team,

I noticed a small mismatch between the parameter counts I get when inspecting the YOLOv26 models and the numbers listed in the documentation.

- yolo26n: measured 2.57M params vs docs 2.4M

- yolo26s: measured 10.1M params vs docs 9.5M

Could you clarify what accounts for this difference?

### Additional

_No response_

### PR DESCRIPTION
**Pull request by raimbekovm**
## Description

Adds a footnote to docs explaining that documented Params/FLOPs are measured on the fused inference model after `model.fuse()`, which merges Conv+BatchNorm layers and, for end2end models, removes the auxiliary one-to-many detection head. Pretrained `.pt` checkpoints retain the full training architecture, so `Model.info()` will report higher counts.

Fixes #23593

## Testing

- `mkdocs build` passes with no new errors
- Verified parameter count differences experimentally (YOLO26n: 6.35%, YOLOv10n: 17.16%, non-end2end models <0.5%)
- Footnote renders correctly on local `mkdocs serve`

## Checklist

- [x] I have read the [Contributing Guide](https://docs.ultralytics.com/help/contributing/) and [CLA](https://docs.ultralytics.com/help/CLA/)
- [x] I have commented `I have read the CLA Document and I sign the CLA` to sign the CLA (required before merge)
- [x] CI passes
- [x] Tests not applicable

## 🛠️ PR Summary

<sub>Made with ❤️ by [Ultralytics Actions](https://www.ultralytics.com/actions)</sub>

### 🌟 Summary
Clarifies that reported Params/FLOPs are measured *after* `model.fuse()` in YOLO26 and task docs to prevent confusion when users compare pretrained checkpoints. 📝

### 📊 Key Changes
- Adds a footnote to `docs/en/models/yolo26.md` explaining Params/FLOPs are for the fused model after `model.fuse()` (Conv+BN fused and auxiliary one-to-many head removed).
- Adds the same clarification to `docs/en/models/yolov10.md` for consistency.
- Updates task pages (`detect.md`, `segment.md`, `pose.md`, `obb.md`) to explicitly define Params/FLOPs as fused-model values, noting pretrained checkpoints may show higher counts.

### 🎯 Purpose & Impact
- Reduces user confusion when Params/FLOPs differ between tables and locally inspected pretrained checkpoints.
- Sets consistent expectations across model and task documentation for end2end architectures and fusion behavior.
- Improves reproducibility and interpretability of published model stats. ✅

### CODE REVIEW

**Comment by UltralyticsAssistant** [comment]
👋 Hello @raimbekovm, thank you for submitting a `ultralytics/ultralytics` 🚀 PR! This is an automated message, and an engineer will assist you soon. To ensure a seamless integration of your work, please review the following checklist:

-✅ **Define a Purpose**: Clearly explain the purpose of your fix or feature in your PR description, and link to any [relevant issues](https://github.com/ultralytics/ultralytics/issues). Ensure your commit messages are clear, concise, and adhere to the project's conventions.
-✅ **Synchronize with Source**: Confirm your PR is synchronized with the `ultralytics/ultralytics` `main` branch. If it's behind, update it by clicking the 'Update branch' button or by running `git pull` and `git merge main` locally.
-✅ **Ensure CI Checks Pass**: Verify all Ultralytics [Continuous Integration (CI)](https://docs.ultralytics.com/help/CI/) checks are passing. If any checks fail, please address the issues.
-✅ **Update Documentation**: Update the relevant [documentation](https://docs.ultralytics.com/) for any new or modified features.
-✅ **Add Tests**: If applicable, include or update tests to cover your changes, and confirm that all tests are passing.
-✅ **Sign the CLA**: Please ensure you have signed our [Contributor License Agreement](https://docs.ultralytics.com/help/CLA/) if this is your first Ultralytics PR by writing "I have read the CLA Document and I sign the CLA" in a new message.
-✅ **Minimize Changes**: Limit your changes to the **minimum** necessary for your bug fix or feature addition. _"It is not daily increase but daily decrease, hack away the unessential. The closer to the source, the less wastage there is."_  — Bruce Lee

For more guidance, please refer to our [Contributing Guide](https://docs.ultralytics.com/help/contributing/). Don't hesitate to leave a comment if you have any questions. Thank you for contributing to Ultralytics! 🚀

**Review by UltralyticsAssistant — APPROVED** [review]
## 🔍 PR Review

<sub>Made with ❤️ by [Ultralytics Actions](https://www.ultralytics.com/actions)</sub>

PR looks clean. Documentation updates are clear, consistent, and don’t introduce issues.

**Comment by Laughing-q** [comment]
@raimbekovm for classify we also fuse conv and bn I think, you can also add for classify(but remove the end2end part)

**Comment by raimbekovm** [comment]
Added in b0d6644 — classify footnote without the end2end part since cls models only fuse Conv+BN.

**Review by Laughing-q — APPROVED** [review]

**Comment by glenn-jocher** [comment]
Thanks—adding the classify footnote (fusion only) makes sense; if you open a follow-up PR for this, please also sign the CLA by commenting `I have read the CLA Document and I sign the CLA` so it’s eligible to merge. For anyone verifying the table numbers locally, make sure you measure after fusion via [`Model.fuse()`](https://docs.ultralytics.com/reference/engine/model/#ultralytics.engine.model.Model.fuse) (which applies the underlying [`BaseModel.fuse()`](https://docs.ultralytics.com/reference/nn/tasks/#ultralytics.nn.tasks.BaseModel.fuse)):

```python
from ultralytics import YOLO

y = YOLO("yolo26n.pt")
y.fuse()
y.model.info()
```

**Comment by raimbekovm** [comment]
The classify footnote is already included in this PR — see `docs/en/tasks/classify.md`.

**Comment by glenn-jocher** [comment]
You’re right—missed that on my end; the classify clarification in `docs/en/tasks/classify.md` (fusion-only) looks good, so no follow-up PR is needed, and it should align with the published [classification task docs](https://docs.ultralytics.com/tasks/classify/).
